audbackend/core/interface/unversioned.py

```diff
@@ -47,6 +48,7 @@ def checksum(
 
         ..
             >>> backend = audbackend.backend.FileSystem("host", "repo")
+            >>> backend.open()
```

Comment:
Maybe use a `with` statement instead? This will safe one line and also makes sure `close()` is called.

Reply:
But if I use a `with` statement here, don't I have to indent all the following lines? This I would like to avoid for the examples.

Reply:
Why?

Reply:
Does the `with` statement not always work like this:

```python
with audbackend.backend.FileSystem("host", "repo") as backend:
    backend.put_file(...)
```

How can I use it to achieve:
```python
>>> with audbackend.backend.FileSystem("host", "repo") as backend
>>> backend.put_file(...)
```
?

Reply:
Or are you referring to why I don't want to indent the following lines?
Because the part where we open the backend is at the moment not visible to a reader of the documentation.
We could of course change this, but I know from `auglib` that users are very afraid of using something that contains a `with` statement ;)

Reply:
Ah sorry, I overlooked that these lines are executed **before** the docstring example and since it is anyway not visible to the user we can of course stay with your solution. Only where we show code to the user we should either use a `with` statement or also call `close()`.

Reply:
OK, makes sense, then let's stay with the current solution here.

Reply:
> But I know from auglib that users are very afraid of using something that contains a with statement

I must admit that I like `with` statements :)